pkg/alert/management/management_controller_test.go

```diff
@@ -0,0 +1,272 @@
+package management_test
+
+import (
+	"context"
+	"testing"
+
+	osmv1 "github.com/openshift/api/monitoring/v1"
+	monv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
+	"github.com/stretchr/testify/assert"
```

Comment:
golang testing + testify id a valid choice. However, please consider that ginkgo + gomega are the non formal standard in the organization and in kubernetes community in general. Using the standard tools will make it easier for others to contribute to the project.

Reply:
in this project, none of the unit tests are using ginkgo/gomega
check for example `pkg/client/client_test.go` or `pkg/client/status_reporter_test.go`

wdyt its better to do?